doc/news/changes/major/20200519DavidWells

```diff
@@ -0,0 +1,7 @@
+New: Added a new class IBTK::FEValues, which is a replacement for libMesh::FE
```

Comment:
Is this a replacement for both codimension 0 and 1 use cases, or just codimension 0?

Reply:
It works in codimension 1 but it cannot yet compute normal vectors so I think the comment is close enough.